sdk/testing/stepwise/environments/docker/runner.go

```diff
@@ -0,0 +1,113 @@
+package docker
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/api/types/network"
+	"github.com/docker/docker/api/types/strslice"
+	docker "github.com/docker/docker/client"
+	"github.com/docker/docker/pkg/archive"
+)
+
+// Runner manages the lifecycle of the Docker container
+type Runner struct {
+	dockerAPI       *docker.Client
+	ContainerConfig *container.Config
+	ContainerName   string
+	NetName         string
+	IP              string
+	CopyFromTo      map[string]string
```

Comment:
This is vague - are these files? Data fields?

Reply:
Deferring and tracking this internally, to follow up in the coming weeks